src/blueprint.test.js

```diff
@@ -369,6 +389,43 @@ module.exports = (test, dependencies) => {
         },
       },
     },
+    'when an implementation has error properties that are extended such as status': (expect) => {
+      const bp = blueprint('sut', {
+        firstName: 'string',
+        lastName: ({ value }) => {
+          const expectedLength = 'World'.length
+          if (typeof value === 'string' && value.length === expectedLength) {
+            return true
+          }
+          const err = new Error(`Must be shorter than ${expectedLength}`)
+          err.status = 400
+          return { err }
+        },
+        fullName: ({ key, value, input, root }) => {
+          const expected = `${input.firstName} ${input.lastName}`
+          if (value === expected) {
+            return { value: true }
+          }
+          const error = new Error(`Full Name should be ${expected}`)
+          error.status = 401
```

Comment:
Why a 401 code here? Just an example?

Reply:
Yeah to be difference than 400 in this unit test